package.json

```diff
@@ -14,11 +14,13 @@
     "fix-lint": "eslint --ext .ts --fix packages/*/src/",
     "test": "lerna run test --concurrency 1",
     "test:unit": "lerna run test:unit --no-bail --concurrency 1",
+    "coverage": "vitest run --coverage",
     "check-readme": "lerna run check-readme",
     "release:publish": "lerna publish from-package --yes --no-verify-access",
     "release:docker": "exit 0"
   },
   "dependencies": {
+    "@flashbots/ethers-provider-bundle": "^0.6.2",
```

Comment:
>"@flashbots/ethers-provider-bundle": "^0.6.2",

delete this dependency from here, we only use it in `executors` package

Reply:
Removed the mentioned dependency from the package.json file